app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb

```diff
@@ -76,17 +76,28 @@ def parse_physical_server(node)
         :model                  => node.model,
         :serial_number          => node.serialNumber,
         :field_replaceable_unit => node.FRU,
-        :computer_system        => {
-          :hardware             => {
-            :networks  => [],
-            :firmwares => [] # Filled in later conditionally on what's available
-          }
-        }
+        # Filled in later conditionally on what's available
+        :computer_system        => {:hardware => {:networks => [], :firmwares => []}}
       }
       new_result[:computer_system][:hardware] = get_hardwares(node)
+      save_host_relationship(node)
       return node.uuid, new_result
     end
 
+    def save_host_relationship(node)
```

Comment:
I don't think it is a work around. I think that is the way this is intended to work.    I believe all we need to do is remove the save! operation in the refresher and then add the :host as a child key in save_physical_server.  This is working for me locally.